src/main/java/Duke.java

```diff
@@ -1,10 +1,84 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+
+    private static final String boundary = "____________________________________________________________";
```

Comment:
For wrapped lines, it is recommended to have 8 indented spaces for the second line onwards